webpack/assets/javascripts/foreman_hosts.js

```diff
@@ -0,0 +1,39 @@
+export let pluginEditAttributes = {
```

Comment:
@parthaa apologies, i was confused by the unusual exports.
more common is to do something like:
`
 const hosts = {
  pluginEditAttributes: {...},
  getAttributesToPost: (selectionType) => {...}
}
export default hosts
`
not a big deal though.